src/core/qgsbookmarkmanager.cpp

```diff
@@ -0,0 +1,510 @@
+/***************************************************************************
+    qgsbookmarkmanager.cpp
+    --------------------
+    Date                 : September 2019
+    Copyright            : (C) 2019 Nyall Dawson
+    Email                : nyall dot dawson at gmail dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsbookmarkmanager.h"
+#include "qgsproject.h"
+#include "qgssettings.h"
+#include "qgssqliteutils.h"
```

Comment:
I think this can go away.

Reply:
Blah ignore, I saw it was needed when importing old sqlite-based bookmarks. I don't understand why github won't let me delete comments in draft mode.